canonical-web-and-design/ubuntu.com
# 7413 appliance governance

Pair coded with @anthonydillon 

## Done

- Updated the /appliance/governance page content
- Added an intitial contact us page (will need to update the details when the real one is created)
- Added the second-level nav for the appliance section

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/governance
- http://0.0.0.0:8001/appliance/contact-us
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)


## Issue / Card

Fixes #7413
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2715
